playbook/actions/ai.go

```diff
@@ -0,0 +1,302 @@
+package actions
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/flanksource/commons/duration"
+	"github.com/flanksource/duty/context"
+	"github.com/flanksource/duty/models"
+	"github.com/flanksource/duty/query"
+	"github.com/flanksource/duty/types"
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+
+	"github.com/flanksource/incident-commander/api"
+	v1 "github.com/flanksource/incident-commander/api/v1"
+	"github.com/flanksource/incident-commander/llm"
+)
+
+type AIAction struct{}
+
+type AIActionResult struct {
+	Logs string `json:"logs,omitempty"` // TODO: only naming this "logs" because the frontend has proper formatted display for this field
```

Comment:
I think we need to return a markdown response, 

We could probably convert  https://codesandbox.io/p/sandbox/markdown-to-slack-converter-iphgn?file=%2Fsrc%2Findex.js%3A25%2C56 into a go function and then just call `markdownToSlackBlocks`